vllm/engine/arg_utils.py

```diff
@@ -201,6 +201,10 @@ def __post_init__(self):
         if not self.tokenizer:
             self.tokenizer = self.model
 
+        # Force prefix caching disable.
+        if envs.VLLM_USE_V1 and envs.VLLM_V1_FORCE_DISABLE_PREFIX_CACHING:
+            self.enable_prefix_caching = False
```

Comment:
Some V0 LoRA tests runs the engine, without-lora, with-lora, without-lora and then again with-lora. Like in https://github.com/vllm-project/vllm/blob/74062740416db8572627dda1f87925268ba2f1d3/tests/lora/test_llama_tp.py#L59 
The comparison requires that prefix caching be disabled. 
I added `VLLM_V1_FORCE_DISABLE_PREFIX_CACHING` to turn off prefix-caching via an environment variable. I believe it could be useful in other tests / benchmarking. 

What do you guys think ? @rickyyx @comaniac @robertgshaw2-neuralmagic @WoosukKwon  

There are not that many existing V0 tests that require this. If this  env variable is undesirable, working around it shouldn't be too hard.

Reply:
Oh can you use `no-enable-prefix-caching` somehow?

Reply:
I'll take a look. Not-using/Removing this env variable shouldn't be too painful. I added it as it looked like a good option to have.

Reply:
I have removed the enviroment variable 👍